lighthouse-core/audits/byte-efficiency/modern-image-formats.js

```diff
@@ -106,21 +131,29 @@ class ModernImageFormats extends ByteEfficiencyAudit {
           naturalHeight,
           naturalWidth,
         });
+        avifSize = ModernImageFormats.estimateAvifSizeFromDimensions({
+          naturalHeight,
+          naturalWidth,
+        });
         fromProtocol = false;
       }
 
-      if (image.originalSize < webpSize + IGNORE_THRESHOLD_IN_BYTES) continue;
+      if (webpSize === undefined || avifSize === undefined) continue;
+
+      const wastedWebpBytes = image.originalSize - webpSize;
```

Comment:
it might be worth a comment somewhere in here that this is deliberate. `// Visible wastedBytes is from AVIF savings, but include wastedWebpBytes in LHR` or whatever